tests/unit/test_gff3_parser.py

```diff
@@ -0,0 +1,102 @@
+"""
+Data-driven tests for rna quantification.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+import ga4gh.gff3 as gff3
+import unittest
+
+
+class TestGff3ParserOnTypicalFile(unittest.TestCase):
+    """
+    Data driven unit tests for the GFF3 parser
+    """
+    def setUp(self):
```

Comment:
Usually with data driven tests we want to compare the results of two difference parsing approaches. VCF is the canonical example: we use PyVCF to parse the VCFs and we make sure our code agrees with it. Can we import some other GFF parser to parse the file for us?